doc/nrf/ug_dev_model.rst

```diff
@@ -0,0 +1,538 @@
+regressions, etc.
+
+.. _dm-user-workflow:
+
+User workflow
+*************
+
+This section describes suggested user workflows to develop and maintain an
+application based on the |NCS|.
+
+The development and distribution model that the |NCS| uses has been described at
+length in the sections above, and the present section deals with the actual
+practicalities of developing an application that is based on the |NCS| from a
+version control and maintenance point of view.
+
+Common Git and west operations
+==============================
+
+In order to maintain an application that uses the |NCS|, it is recommended
+to use the same tools that Nordic employs to develop it. In particular and
+when it comes to version control, a combination of `Git`_ and
+:ref:`west<zephyr:west>` is used to interact with the Git repositories
+that make up the |NCS|.
+
+As previously described, the |NCS| contains the repositories listed below, along
+with the tool that is used to manage them:
+
+* The manifest repository, `fw-nrfconnect-nrf`_. This repository is managed by
+  the user using Git exclusively, since west will not modify or update it in
+  any way. The exception to this is the ``west init`` command, which is able to
+  clone the manifest repository automatically at an arbitrary revision.
+
+* The west projects. Those are listed in the manifest repository's
+  :file:`west.yml` manifest file. They are entirely managed by west, which will
+  clone them or check out a specific revision of them every time ``west update``
+  is run by the user.
+
+There are several basic operations that |NCS| users will perform routinely with
+Git and west, the most common of which are described in the subsections below.
+
+.. _dm-wf-get-ncs:
+
+Obtain a copy of the |NCS|
+--------------------------
+
+In order to obtain a fresh copy of the |NCS| at revision ``{revision}`` and
+place it in a folder named :file:`ncs`, use the following commands::
+
+  west init -m https://github.com/NordicPlayground/fw-nrfconnect-nrf --mr {revision} ncs
+  cd ncs
+  west update
+
+Replace ``{revision}`` with any revision you wish to obtain. This can be
+``master`` if you want the bleeding edge or any released version (eg.
+``v1.0.0``). If you omit the ``--mr`` parameter, west will default to
+``master``.
+
+.. _dm-wf-update-ncs:
+
+Update a copy of the |NCS|
+--------------------------
+
+If you already have a copy of the |NCS| and wish to update it or switch to a new
+revision, then you only need to do the following::
+
+  cd ncs/nrf
+  git fetch {remote}
+  # Check out the latest master branch
+  git checkout {remote}/master
+  # or check out a release
+  git checkout {revision}
+  west update
+
+Where ``{remote}`` is the Git remote that points to the official Nordic
+repository. Typically this is called ``origin`` but may have been renamed. You
+can use ``git remote -v`` to list all your remotes.
+Note that using ``git checkout`` is one of the multiple ways of achieving this.
```

Comment:
Another single newline here, and in a few other places -- it's fine if they're intentional, they just look weird so I wanted to check if that's what you meant.

Reply:
Yep, another oversight! Thanks again!